core/src/main/java/de/eldoria/bloodnight/mobs/MobRegistry.java

```diff
@@ -33,36 +31,22 @@ public MobRegistry() {
      * @throws MobIdAlreadyTakenException when the id is already taken.
      */
     public void add(CustomMob mob) {
-        check(mob);
-        if (mobs.containsKey(mob.id())) throw new MobIdAlreadyTakenException(mob, get(mob.id()));
-        mobs.put(mob.id(), mob);
-    }
-
+        Checks.notNull(mob, "Mob can not be null");
 
-    /**
-     * Returns the {@link CustomMob} associated with the given id.
-     *
-     * @param id the id of the mob to be retrieved.
-     * @return the {@link CustomMob} associated with the given id, or null if no mob is found.
-     */
-    public CustomMob get(String id) {
-        return mobs.get(id);
```

Comment:
Why remove the get method?

Reply:
It is api and most probably just there for completeness of the provided api. It doesnt has to be used.
Please dont just delete stuff that is unused. I most probably put it there for a reason and writing docs again is a pain.